experiments/ext_api/word_count_demo/requirements.txt

```diff
@@ -1,3 +1,4 @@
 kafka-python
 redis
 boto
+boto3
```

Comment:
Is there a reason we need two versions of boto here?

Reply:
The kinesis integration is currently using boto not boto3. I just need to update it...